config/site.yml.tmpl

```diff
@@ -25,6 +25,11 @@ secure_cookies: false
 #
 secret_token: "change-me"
 
+# By default, tell Rails to not serve static assets. The best practice is to
+# have a proxying web server such as Apache or Nginx serve static assets
+# (images, stylesheets, javascript) for you. Change this to 'true' if you want
+# Rails to be responsible for serving the static assets
+serve_static_assets: false
```

Comment:
For new installs, wouldn't we always want this to be set to something vs. relying on a fallback? Or is it that commenting this out means that the user doesn't have to worry about changing it?